PSKoans/Koans/Foundations/AboutLoopsAndPipelines.Koans.ps1

```diff
@@ -74,8 +74,19 @@ Describe 'Pipelines and Loops' {
         }
         $Values | Should -Be @(0, 1, 2, 3, 4)
 
-        $Values = while ($true) { # watch out for infinite loops!
-            Write-Output (++$Tick) # Remember: an undeclared variable acts as zero until we increment it!
+        $Values = while ($true) {
+            # watch out for infinite loops!
+
+            # Remember: an undeclared variable acts as zero until we increment it!
+            # Also, to make sure it is returned the value must be wrapped in brackets
+            # else the variable is simply output and not returned.
```

Comment:
This is ambiguous; we want the variable value to be output, after all! To powershell, output and 'return' are often synonymous.

Perhaps something more like... "Incrementing or assigning a value to a variable won't output the value unless the expression is wrapped in parentheses." 

Or something along those lines!

Reply:
facepalm moment. I mean't to write "else the variable is simply assigned and not output." but I was talking to someone at the time and wrote it wrong. I've added your description and squashed my commits. I don't know if you squash or not

Reply:
I don't github much, so that is gibberish to me. I'll read up on it later :)